torch/testing/_internal/common_methods_invocations.py

```diff
@@ -628,6 +632,12 @@ def sample_inputs_add(op_info, device, dtype, requires_grad, **kwargs):
                                                requires_grad=requires_grad),
                                    args=(arg,),
                                    broadcasts_input=broadcasts_input))
+        if kwargs.get("use_alpha", False):
```

Comment:
This is a cool idea but let's only add one test case when alpha is passed like this:

```
# Adds an extra sample using "alpha" if it's passed in kwargs
if 'alpha' in kwargs:
  a = make_tensor((S, S, S), device=device, dtype=dtype, requires_grad=requires_grad)
  b = make_tensor((S, S, S), device=device, dtype=dtype, requires_grad=requires_grad)
  sample = SampleInput(a, args=(b,), kwargs={'alpha': kwargs['alpha']})
  samples.append(sample)
```

Reply:
Just because alpha is so rarely used, this should sanity-check it without doubling test time